WordPress/gutenberg
# RangeControl: Add support for large 40px number input size

## What?

Add support for the `size="__unstable-large"` prop within the RangeControl component, to support the effort towards consistent components across the board re https://github.com/WordPress/gutenberg/issues/46734. 

Related to https://github.com/WordPress/gutenberg/pull/49101 and https://github.com/WordPress/gutenberg/issues/46741.

## Why?
Support the consistent component effort in https://github.com/WordPress/gutenberg/issues/46734.

## How?
1. Add a size prop. 
2. Tweak styling to align:center from start (I don't think this changes the component's layout for default sizes)
3. Zero out margin bottom, as it was not always zeroed—like when I tested in https://github.com/WordPress/gutenberg/pull/49101.
4. Set width of large input to `size(20)`.
5. Add `size="__unstable-large"` to the Cover block's Overlay Opacity control for testing. 

## Testing Instructions

1. Open a Post or Page.
6. Insert a Cover block.
7. Add an image or pick a color.
8. Open Block Inspector.
9. Select the "Styles" tab.
10. See the larger 40px input on the "Overlay Opacity" RangeControl.

## Screenshots or screencast 

Before: 

<img width="280" alt="CleanShot 2023-03-15 at 11 44 24" src="https://user-images.githubusercontent.com/1813435/225363866-be2f49ee-78f4-4444-abc0-b56368fb3c60.png">

After: 

<img width="280" alt="CleanShot 2023-03-15 at 12 42 23" src="https://user-images.githubusercontent.com/1813435/225379907-3fcd9698-aa1f-4360-b47f-be78b02d536f.png">
